libs/@blockprotocol/graph/src/graph-module.json

```diff
@@ -131,10 +129,14 @@
         "properties": {
```

Comment:
Too far from the diff but can we update the description of this message to be
"Request to retrieve a subgraph rooted at a specific entity"

Reply:
[Update description for getEntity](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/d132aa2ecfa566c6c7d1bd854b6330aad06942d9)